### PR TITLE
chore: bump nuxt to 3.7

### PR DESCRIPTION
**Pull request by connoratrug**

### CODE REVIEW

File: apps/nuxt3-ssr/package.json
Comment (mswertz) [inline apps/nuxt3-ssr/package.json:31]: renovate doesn't like ^ because then the actual build version is not deterministic. Did you do this with a reason?

**Review by mswertz — CHANGES_REQUESTED** [review]

File: apps/nuxt3-ssr/package.json
Comment (connoratrug) [reply apps/nuxt3-ssr/package.json:31]: nuxt has its own preferred update system ( call) it uses the default ( with the ^ ) , ill change it

**Review by mswertz — APPROVED** [review]

File: apps/nuxt3-ssr/package.json
Comment (connoratrug) [reply apps/nuxt3-ssr/package.json]: i'll wait for ci build and check the preview before merging

**Pull request merged**
